deployer/src/algolia_internal_api.py

```diff
@@ -19,7 +19,6 @@ def get_headers():
     admin_api_key = environ.get('API_KEY_PROD').encode()
     auth_token = b64encode(app_id + b":" + admin_api_key).decode().replace('=',
                                                                            '').replace(
-
```

Comment:
unrelated to the commit, please undo

Reply:
✅

Reply:
I don't understand why you are solving this comment: I asked you to undo the change, but it is still there. Please don't modify the layout of this file in this commit.